tests/unit/utils/parsing_test.py

```diff
@@ -130,3 +166,98 @@ def test_parse_eyelink_raises_value_error(tmp_path, patterns):
     expected_substrings = ['invalid pattern', '1']
     for substring in expected_substrings:
         assert substring in msg
+
+
+@pytest.mark.parametrize(
+    'metadata, expected_version, expected_model',
+    [
+        pytest.param(
+            '** VERSION: EYELINK II 1\n'
+            '** EYELINK II CL v6.12 Feb  1 2018 (EyeLink Portable Duo)',
+            '6.12',
+            'EyeLink Portable Duo',
+            id='eye_link_portable_duo',
+        ),
+        pytest.param(
+            '** VERSION: EYELINK II 1\n'
+            '** EYELINK II CL v5.12 Feb  1 2018',
+            '5.12',
+            'EyeLink 1000 Plus',
+            id='eye_link_1000_plus',
+        ),
+        pytest.param(
+            '** VERSION: EYELINK II 1\n'
+            '** EYELINK II CL v4.12 Feb  1 2018',
+            '4.12',
+            'EyeLink 1000',
+            id='eye_link_1000_1',
+        ),
+        pytest.param(
+            '** VERSION: EYELINK II 1\n'
+            '** EYELINK II CL v3.12 Feb  1 2018',
+            '3.12',
+            'EyeLink 1000',
+            id='eye_link_1000_2',
+        ),
+        pytest.param(
+            '** VERSION: EYELINK II 1\n'
+            '** EYELINK II CL v2.12 Feb  1 2018',
+            '2.12',
+            'EyeLink II',
+            id='eye_link_II',
+        ),
+        pytest.param(
+            '** VERSION: EYELINK REVISION 2.00 (Aug 12 1997)',
+            '2.00',
+            'EyeLink I',
+            id='eye_link_I',
+        ),
+        pytest.param(
+            '** VERSION: nothing\n',
+            'unknown',
+            'unknown',
+            id='unknown_version_1',
+        ),
+        pytest.param(
+            '** VERSION: EYELINK II 1\n'
+            '** EYELINK II CL Feb  1 2018 (EyeLink Portable Duo)',
+            'unknown',
+            'unknown',
+            id='unknown_version_2',
+        ),
```

Comment:
Please add another test case where there is metadata but no version information, for example like this:

```
** DATE: Wed Mar  8 09:25:20 2023
** TYPE: EDF_FILE BINARY EVENT SAMPLE TAGGED
```

which should have both model and version parsed as `unknown`